src/server/utils/onModuleLoad.js

```diff
@@ -98,6 +110,7 @@ export default function onModuleLoad({
   }
 
   const serverStateConfig = getServerStateConfig();
+
```

Comment:
Why add this?

Reply:
Um, I think this is just a mistake haha that's odd?